src/QuadForm/Torsion.jl

```diff
@@ -2159,3 +2160,79 @@ underlying abelian group is in Smith normal form.
 """
 is_snf(T::TorQuadModule) = is_snf(abelian_group(T))
 
+################################################################################
+#
+#  Isotropic check
+#
+################################################################################
+
+@doc raw"""
+    is_totally_isotorpic(T::TorQuadModule)
+
+Return whether the quadratic form on the torsion quadratic module `T` vanishes.
+"""
+function is_totally_isotropic(T::TorQuadModule)
+  for a in gens(T)
+    if !is_zero(quadratic_product(a))
+      return false
+    end
+    for b in gens(T)
```

Comment:
```suggestion
    for b in gens(T)
      b == a && continue
```
we do not necessarily need `a` to be orthogonal to itself (for the bilinear form). Otherwise, in that case you would consider only degenerate modules